Additional guidance on self-promotional materials, with examples (#1926)
writethedocs/www: docs/slack.rst

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

You're welcome to share, link to, and mention your own work, including blog posts, social media, events, and paid products or services.

> That said, posts where you stand to benefit from others' money, personal information, click-throughs, or labor come with higher expectations.

Reviewer: I'm not 100% sure what this means. Maybe this?

```suggestion
That said, posts where you stand to benefit from others' money, personal information, attention, or labor come with higher expectations.
```

Author: click-throughs = ad revenue that is click/impression based. But I think 'attention' works there.

  Clearly explain what you are promoting, why it matters to documentarians, and your involvement.
  Be up front about paywalls, registration requirements, referral benefits, or commissions.
  If you're running a survey, say when and how the results will be published.

> 

Reviewer: These spaces don't appear elsewhere. Do you want to add them to _all_ the bulleted lists, or none of them? (My kingdom for a Prettier-like formatter for reStructuredText.)

Author: Lets do none for consistency.

  Too much self-promotion creates distrust and hostility.
  If you're not sure if it's too much, aim to help others 10 times for every time you self-promote.

- **Post publicly instead of sending direct messages (DMs)**.
  A DM is like a home and you're like a vampire: you're not allowed in without an invitation.
  It can be tempting to DM people, but please respect their inboxes and post in public channels unless invited to DM.

- **Read the room.**
  The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
  If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
  If you're asked to stop, then stop.

- **Treat the community like friends, not customers.**
  Posts that resemble infomercials are harder for the community to engage with and for the moderation team to distinguish from spam.
  If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
  Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.

> Here are some examples of common self-promotion situations and how to self-promote responsibly (and irresponsibly):

Reviewer 2: ```suggestion
Here are some examples of common self-promotion situations, and some guidelines on how to self-promote responsibly:
```

> Here are some examples of common self-promotion situations, and some guidelines on how to self-promote responsibly:

Reviewer: This is a large enough addition to section that takes up the full vertical height of the screen (at least on my screen), which makes it somewhat harder to skim. Maybe this section could be broken in three:

- The first bit with the actual guidelines, immediately following the heading
- A new subhead (h4) for the examples
- A new subhead (h4) for the self-promotion channel recommendations

(Or if that doesn't suit, I can suggest cutting down the text a bit.)

Author: Done!

You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion

>   **DO** share it in #community-showcase and tell us why you think it would be interesting or relevant to documentarians

Reviewer: Channel names are `**#channel-name**` on the rest of the page, so I'd assume we should use the same convention here (this would need correcting throughout this section)

```suggestion
  **DO** share it in **#community-showcase** and tell us why you think it would be interesting or relevant to documentarians
```

Author: Done manually

>   **DON'T** cross-post that content across multiple channels

Reviewer: This is formatted as a description list and I'm not sure that's the right choice. With a screen reader it's a bit weird (e.g., VoiceOver says "description list n items") but serviceable, but visually there are no list markers, so it's it's unclear where the list ends (i.e., "Here are some channels" might be another example).

These entries might just work as regular paragraphs. That is:

> Description of an example. **Do** x. **Don't** do y.

Author: And done

Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event)

>   **DO** share it in #community-events and tell others your connection to the event

Reviewer: Bold and shouting is more emphasis than other instruction on the page, which suggests that these are the most important instructions on the page. I don't think that's quite right. Maybe shout or bold, but not both?

Author: Done

>   **DON'T** use "marketing language" to obscure the purpose of the event, your relationship to the employer, or the costs associated to attendance

Reviewer: I don't know what "marketing language" is and I'd have a hard time citing this in a moderation action, but that latter part seems like the real, substantive issue.

```suggestion
  **DON'T** obscure the purpose of the event, your relationship to the employer, or the costs associated to attendance
```

> Someone else has asked a question and you're the author of an article, blog, book, or similar resource that answers the question
>   **DO** reply by briefly summarizing your article, noting your authorship, and linking to the article

Reviewer: Consistency with example given

```suggestion
Someone else has asked a question and you're the author of an article, blog, book, or similar resource that answers the question
  **DO** reply by briefly summarizing your work, noting your authorship, and linking to the resource
```

>   **DON'T** solicit views/clickthroughs/purchases of your content, especially via Direct Messages or cross-posting out-of-context

Reviewer: I'm not sure how "soliciting" relates to the example. As written it reads to me as: do tell people about a book you've read, but don't do it in a way that suggests they ought to read it. It does sound like you're touching on something about germaneness though, which I like. What do you think of this?

```suggestion
  **DON'T** cross-post a reply outside the thread or channel or send the person asking the question a direct message
```

> You are offering cash for gold, selling diet supplements, representing the estate of a recently diseased and heretofore unknown family member, or doing some other activity unrelated to documentation but for some reason requires our members to DM you their bank account information
>   **DO** consider better ways to engage with this community
>   
>   **DON'T** post that content to Write the Docs
> 

Reviewer: OK, I realize this joke was my suggestion but seeing it written out in context, I think we might need to kill our darlings here. Nobody who has this intent is going to read the guidelines and we're just going to delete it anyway. I suggest that this doesn't actually serve a plausible audience. 😞

```suggestion

```

Reviewer 2: :-D but it goes give you something to link to when you do delete it. :-D

Author: I cry

  If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
  Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.

> Common Self-Promotion Situations

Reviewer: Consistency nit

```suggestion
Common self-promotion situations
```

++++++++++++++++++++++++++++++++

Here are some examples of common self-promotion situations and how to self-promote responsibly (and irresponsibly):

- You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion.

>   **Do** share it in **community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.

Reviewer: consistency nit

(I mean, I guess you could go the other way and change all the channel mentions to no `#`, but I think it's not obviously a channel otherwise)

```suggestion
  **Do** share it in **#community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.
```

>   **Don't** cross-post that content across multiple channels or use the **community-showcase** (or any other) channel for link-farming or drive-by engagement only.

Reviewer: consistency nit

```suggestion
  **Don't** cross-post that content across multiple channels or use the **#community-showcase** (or any other) channel for link-farming or drive-by engagement only.
```

- Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event).

  **Do** share it in #community-events and tell others your connection to the event.

  **Don't** obscure the purpose of the event, your relationship to the employer, or the costs associated to attendance.

- Someone else has asked a question and you're the author of an article, blog, book, or similar resource that answers the question.

  **Do** reply by briefly summarizing your work, noting your authorship, and linking to the resource.

  **Don't** cross-post a reply outside the thread or channel or send the person asking the question a direct message.

> - A moderator has raised one or more of your posts as violating the Self-Promotion Guidelines, and provided you with feedback and guidance.

Reviewer: style nit

Since we use sentence case, there are no capital-G guidelines

```suggestion
- A moderator has raised one or more of your posts as violating the self-promotion guidelines, and provided you with feedback and guidance.
```

  **Do** read the guidance, review our rules, and modify your posting patterns to respect the Write the Docs community.

>   **Don't** use DMs, alt-accounts, "clever" interpretations of the rules/guidance, or other methods to ignore or "get around" the Moderator feedback.

Reviewer: consistency nit + a c'mon own it

I say use the informal language with confidence.

```suggestion
  **Don't** use DMs, alt-accounts, clever interpretations of the rules/guidance, or other methods to ignore or get around the moderator feedback.
```

> Suggested Channels for Self-Promotion

Reviewer: consistency nit

```suggestion
Suggested channels for self-promotion
```

+++++++++++++++++++++++++++++++++++++

Here are some channels specifically for wholesome self-promotion:

> - **job-posts-only** is the best place to post paid jobs.
> - **freelance** is the best place to post freelance projects.
> - **community-events** (formerly known as `#calls-for-papers`) is a good place to announce non-Write the Docs events.
> - **community-help-wanted** is a good place to share surveys, calls for research interview subjects, and calls for participation on open-source projects.
> - **community-showcase** is a space for responsible self-promotion.

Reviewer: More consistency nits

```suggestion
- **#job-posts-only** is the best place to post paid jobs.
- **#freelance** is the best place to post freelance projects.
- **#community-events** (formerly known as `#calls-for-papers`) is a good place to announce non-Write the Docs events.
- **#community-help-wanted** is a good place to share surveys, calls for research interview subjects, and calls for participation on open-source projects.
- **#community-showcase** is a space for responsible self-promotion.
```

  If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
  Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.

> For example, a link to your own Hashnode or Medium article without being related to an ongoing conversation is Self-Promotional.

Reviewer 3: ```suggestion
For example, a link to your own Hashnode or Medium article without being related to an ongoing conversation is self-promotional.
```

> Similarly, a post to a virtual meetup or "Office Hours" setting where the purpose is evangelizing your own product(s) is Self-Promotional (even if it's your companies work and not your own!).

Reviewer 3: ```suggestion
Similarly, a post to a virtual meetup or "Office Hours" setting where the purpose is evangelizing your own product(s) is self-promotional (even if it's your company's work and not your own!).
```

Author: you would not believe how many times i went back and forth on pluralization here. blah.

But if there is an ongoing conversation on a topic you've written or created content for, you can offer to post that content and do so if the community expresses interest.

> 

Reviewer: A couple of thoughts here:

Do we think people are confused by what counts as self-promotion? That is, we do need to explicitly call out these things as self-promotion? (I haven't had push back from folks—i.e. disagreement that something is self-promotion—but that's just my experience.)

Also, I really like the idea of using examples to help people understand what to do. What do you think of building this out into a little playbook? Something along these lines?

> Here are some examples of common self-promotion situations and how to self-promote responsibly:
> 
> - You've written a blog post that you want to share with the community, but it's not closely related to an ongoing discussion: share it in **#community-showcase** and tell us why you think it would be interesting to documentarians.
> - Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event): share it in **#community-events** and tell others your connection to the event.
> - Someone else has asked a question and you're the author of an article that answers the question: reply by briefly summarizing your article, noting your authorship, and linking to the article.

And maybe…

> - You're offering cash for gold, selling diet supplements, representing the estate of a recently deceased and heretofore unknown family member, or doing some other activity unrelated to documentation: do not post.

Author: No pushback yet, but I generally think the examples as guidance is nice - it leaves things less nebulous as to why one thing might be OK and another gets ``:spam:``

And i like all of your suggestions :) Will implement shortly.

Reviewer 3: I love these suggestions! I've had one guy get mad at me and swear up and down that people would recognize who he was and know that he was a valued community member who would _never_ advertise to them...and I'd never heard of him before 😂 Other than that, people tend to be pretty reasonable, and these guidelines spell it out really nicely!

  If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
  Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.

Common self-promotion situations
++++++++++++++++++++++++++++++++

Here are some examples of common self-promotion situations and how to self-promote responsibly (and irresponsibly):

- You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion.

  **Do** share it in **#community-showcase** and tell us why it is relevant to the practice of writing, creating, maintaining, or building docs.

  **Don't** cross-post that content across multiple channels or use the **#community-showcase** (or any other) channel for link-farming or drive-by engagement only.

- Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event).

>   **Do** share it in #community-events and tell others your connection to the event.

Reviewer: oh no I missed one

```suggestion
  **Do** share it in **#community-events** and tell others your connection to the event.
```

- **#community-help-wanted** is a good place to share surveys, calls for research interview subjects, and calls for participation on open-source projects.
- **#community-showcase** is a space for responsible self-promotion.

> Please note that we will remove posts in those channels if they violate our Slack rules.
> 

Reviewer: (style, optional)

Could crib from the preceding section, for consistency?

```suggestion
Posts in those channels not meeting the posting rules may result in :ref:`moderation action <moderation-action>`, such as a warning or deletion.

```
